holidays/countries/austria.py

```diff
@@ -86,6 +102,46 @@ def _populate_bank_holidays(self):
         # New Year's Eve.
         self._add_new_years_eve(tr("Silvester"))
 
+    def _populate_subdiv_1_bank_holidays(self):
+        # Patron saint of Burgenland: Hl. Martin von Tours (St. Martin of Tours)
+        self._add_holiday_nov_11(tr("Hl. Martin"))
+
+    def _populate_subdiv_2_bank_holidays(self):
+        # Patron saint of Carinthia: Hl. Josef (St. Joseph's day)
```

Comment:
I suggest simplifying the comments to just their names in English for brevity.
```suggestion
        # St. Joseph's Day.
```